CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ release.
   ([#402](https://github.com/open-telemetry/semantic-conventions/pull/402))
 - BREAKING: Change `network.type` from recommended to opt-in in HTTP semconv.
   ([#410](https://github.com/open-telemetry/semantic-conventions/pull/410))
+- BREAKING: Remove `thread.daemon`, and introduce `jvm.thread.daemon` instead.
```

Comment:
Nit: Maybe now that we have proper sections in here, we can drop the redundant "BREAKING" prefix.